src/block_ffm.rs

```diff
@@ -389,13 +390,14 @@ impl<L: OptimizerTrait + 'static> BlockTrait for BlockFFM<L> {
         further_blocks: &[Box<dyn BlockTrait>],
         fb: &feature_buffer::FeatureBuffer,
         pb: &mut port_buffer::PortBuffer,
+        mask_interactions: bool,
     ) {
         debug_assert!(self.output_offset != usize::MAX);
 
         let num_outputs = (self.ffm_num_fields * self.ffm_num_fields) as usize;
         let myslice = &mut pb.tape[self.output_offset..(self.output_offset + num_outputs)];
         myslice.fill(0.0);
-
+        let mut rng = rand::thread_rng();
```

Comment:
Should we seed this generator?

Reply:
I don't think we should seed, because we want at least each forward call to have a different random number.. i could pass the index to the forward pass function, but thought it was an overkill.. 
in any case, in the tests I'm not checking for an specific number, I'm just checking if it's close to the single prediction